tests/utils/minio.go

```diff
@@ -321,7 +321,7 @@ func MinioDefaultClient(namespace string) corev1.Pod {
 			Containers: []corev1.Container{
 				{
 					Name:  "mc",
-					Image: "minio/mc:RELEASE.2021-04-22T17-40-00Z",
+					Image: "minio/mc:RELEASE.2022-06-11T21-10-36Z",
```

Comment:
What about putting this value in a constant on top of the source file? It would be more evident.

Reply:
yeah, I can do that.